tester/dns/tester.go

```diff
@@ -19,9 +19,10 @@ import (
 
 // Tester is a load tester for DHCPv6
```

Comment:
whoops, there is a typo here :D it should be DNS :) do you mind changing it while you are at it?